personality/voms/files/tomcat5.conf

```diff
@@ -0,0 +1,12 @@
+CATALINA_HOME="%s"
```

Comment:
Definitely ncm-metaconfig, but this file also doesn't seem to be referenced anywhere in this PR.